src/main/java/ChattyChatBot.java

```diff
@@ -0,0 +1,132 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class ChattyChatBot {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    private static final String BOT_NAME = "Chatty Chat Bot";
+
+    private static final String BYE_STRING = "bye";
+    private static final String LIST_STRING = "list";
+    private static final String DONE_STRING = "done";
+    private static final String TODO_STRING = "todo";
+    private static final String DEADLINE_STRING = "deadline";
+    private static final String EVENT_STRING = "event";
+
+    private static final String SPACE_SEPARATOR = " ";
+    private static final String DOT_CHARACTER = ".";
+    private static final String BY_STRING = "/by";
+    private static final String AT_STRING = "/at";
+
+    private static final String LINE_BREAK = "____________________________________________________________";
+
+    private static final String ADDED_TASK_CONFIRMATION = "Got it. I've added this task:";
+    private static final String TASK_SUMMARY_FIRST_HALF = "Now you have ";
+    private static final String TASK_SUMMARY_SECOND_HALF = " tasks in the list.";
```

Comment:
I was thinking that all these LoCs could be put into a separate enum Java file. Other than that, declaring those constants are nice

Reply:
Thanks for the advice! I've refactored the constants into `Constants.java`